index.html

```diff
@@ -520,32 +520,22 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/CristianBahnaru.jpeg" alt="Poza Cristian Bahnaru" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Cristian Aurelis Bahnaru</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    I work well with others and value their presence and contributions when working on collaborative projects. After all, we all need to be able to work together to succeed.   I am also very approachable and always willing to help and support my coworkers when needed..I have an passion for Web Development and programming languages, learn new technologies and improve my knowledge.
                   </p>
-                  <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                   <p>My favorite motto is :</p>
+                      <p>"Inspire people,Empower people,Share Vision,Lead Change,Make it Happen"</p>
```

Comment:
There is no need for 2 `<p>` tags here. If you want to emphasize the motto (make it italic), you can use the `<em>` tag.

Reply:
Also, put a space after the comma, and do not use capital letters after the comma. Finish the paragraph with a full stop.